compiler/damlc/tests/src/DA/Test/Repl/FuncTests.hs

```diff
@@ -327,6 +327,15 @@ functionalTests replClient replLogger serviceOut options ideState = describe "re
           , input ":json x"
           , matchOutput "1"
           ]
+    , testInteraction' "collision"
+      -- Test that collisions are handled correctly if users qualify names.
+          [ input "let x = ReplTest.NameCollision \"abc\""
+          , input "x"
+          , matchOutput "NameCollision {field = \"abc\"}"
+          , input "y <- pure $ Colliding.NameCollision 42"
+          , input "y"
+          , matchOutput "NameCollision {field = 42}"
```

Comment:
Not for this PR, but I'm wondering if the outputs should be qualified too in this case.

Reply:
Not sure I understand what you are suggesting. This is just the `Show` instance. We don’t have any control over this and it doesn’t depend on imports.

Reply:
Ah, right, forgot that this is just `Show`. Nvm.